### PR TITLE
Create stem-audio-table-private-keys.yaml

### PR DESCRIPTION
**Pull request by gy741**
Hello

Added private key exposure detection rule in STEM Audio Table.

This is similar to the existing rules.

If you want, you can add it there.

ref: https://github.com/projectdiscovery/nuclei-templates/blob/master/exposures/configs/server-private-keys.yaml

Thanks.

### CODE REVIEW

**Comment by gy741** [comment]
Note that this equipment has an RCE risk.

I will analyze and add when I have time.

Currently, I've added easy-to-test samples among them.

**Comment by geeknik** [comment]
>
> This is similar to the existing rules.
> 

I think this is perfectly find being a standalone template, especially if this is a sign of things to come from this line of products. 👍🏻

**Review by geeknik — APPROVED** [review]

File: iot/stem-audio-table-private-keys.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - method: GET
+    path:
+      - "{{BaseURL}}/cgi-bin/privatekey.pem"
+
+    matchers-condition: and
+    matchers:
+      - type: word
+        words:
+          - "BEGIN RSA PRIVATE KEY"
+        condition: or
```
Comment (geeknik) [inline iot/stem-audio-table-private-keys.yaml:20]: the `condition: or` appears to be unnecessary as there is only one matcher.

**Review by ehsandeep — APPROVED** [review]

**Comment by ehsandeep** [comment]
thank you @gy741 for adding this.

**Pull request merged**